tests/manage/rgw/test_pods_restart_count.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+from ocs_ci.framework.pytest_customization.marks import tier1
+from ocs_ci.ocs.resources.pod import get_rgw_pods
+from ocs_ci.ocs import constants, defaults
+from ocs_ci.ocs.ocp import OCP
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+def test_rgw_restart_counts():
```

Comment:
please check tests.conftest.verify_rgw_restart_count_fixture, maybe we can use this teardown fixture in some of the tier1 tests, instead of having an explicit test case